mysql-test/suite/wsrep/t/wsrep-recover.test

```diff
@@ -0,0 +1,75 @@
+#
+# Verify that the wsrep XID gets updated in InnoDB rollback segment
+# properly and can be recovered with --wsrep-recover
+#
+
+--source include/have_wsrep.inc
+--source include/have_innodb.inc
+--source include/have_wsrep_provider.inc
+
+CREATE TABLE t1 (f1 INT PRIMARY KEY) ENGINE=InnoDB;
+INSERT INTO t1 VALUES (1);
+
+#
+# Binlog option for recovery run. This must be set in the test because
+# combinations file causes log-bin option to be set from command line,
+# not via my.cnf.
+#
+--let $log_bin = `SELECT @@log_bin`
+if ($log_bin) {
+--let $binlog_opt = --log-bin
+}
+
+#
+# Kill server and run recovery step
+#
+--source include/kill_mysqld.inc
+
+--exec $MYSQLD --defaults-group-suffix=.1 --defaults-file=$MYSQLTEST_VARDIR/my.cnf --log-error=$MYSQL_TMP_DIR/galera_wsrep_recover.log --innodb --wsrep-recover $binlog_opt --core-file > $MYSQL_TMP_DIR/galera_wsrep_recover.log 2>&1
```

Comment:
Would it be possible to reuse `suite/galera/include/galera_wsrep_recover.inc`, and then extract seqno from `$galera_wsrep_start_position`?

Reply:
`galera_wsrep_recover.inc` should be modified a bit to accept binlog option. It can maybe be done as a final step, but I'd rather keep this suite self contained.